modules/sdk-coin-trx/src/trx.ts

```diff
@@ -707,7 +720,7 @@ export class Trx extends BaseCoin {
 
     // a sweep potentially needs to pay for multi-sig transfer, destination account activation and bandwidth
     // TRON foundation recommends 2.1 TRX for guaranteed confirmation
-    if (!recoveryAmount || SAFE_TRON_TRANSACTION_FEE > recoveryAmount) {
+    if (!recoveryAmount || SAFE_TRON_TRANSACTION_FEE >= recoveryAmount) {
```

Comment:
Looks like it will always leave 1 unit behind

Reply:
Yes, that's how Tron works with us currently. If balance is the reserved fee, then remaining is 0, so nothing to sweep